helpers/epinio/epinio.go

```diff
@@ -0,0 +1,36 @@
+package epinio
```

Comment:
isn't this helper only used in tests? Maybe we should put it under `acceptance/` to make it clear.

Reply:
Also, if we are introducing a helper, shouldn't it be re-used here: https://github.com/epinio/epinio/blob/main/acceptance/testenv/epinio.go#L100 ?

Reply:
Done. Moved to the acceptance folder. Initially, I thought of re-using the testenv for these tests. But I decided not to since testenv is explicitly used only for creating testing environments and this package I created is used inside the tests.